README.md

```diff
@@ -16,11 +15,11 @@
 
 ## Challenges:
 
```

Comment:
It would be nice to add a overview below the Challenges Header

Reply:
done